tools/org.eclipse.n4js.hlc.base/src/org/eclipse/n4js/hlc/base/N4jscBase.java

```diff
@@ -381,6 +381,15 @@ public SuccessExitStatus doMain(String... args) throws ExitCodeException {
 				return SuccessExitStatus.INSTANCE;
 			}
 
+			// Make sure the srcFiles are valid
+			for (File srcFile : srcFiles) {
+				if (!srcFile.isFile() && !srcFile.isDirectory()) {
```

Comment:
reuse `org.eclipse.n4js.hlc.base.HlcFileUtils.isExistingReadibleFile(File)`

Reply:
Unfortunately, the method throws `RuntimeException` which is a bit ugly to reuse in this case...